extensions/graphql/src/__tests__/fixtures/graphql-resolvers/recipe/recipe.resolver.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright IBM Corp. 2021. All Rights Reserved.
+// Node module: @loopback/graphql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {query, resolver} from '../../../../';
+import {Recipe} from './recipe.model';
+
+@resolver(of => Recipe)
+export class RecipeResolver {
+  constructor() {}
+
```

Comment:
Can you add tests for Mutations also?

Reply:
That's a great callout- yes, I will add a mutation to ensure the resulting spec is correct.